Assets/Scripts/MiniGames/Pirates/CardDropZone.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CardDropZone : MonoBehaviour
+{
+	private void OnTriggerEnter2D(Collider2D collision) 
+	{
```

Comment:
we're using first bracket at the end of the line style for functions. when you get some time later, try and match that style:
```
private void OnTriggerEnter2D(Collider2D collision) {
	. . .
}
```